### PR TITLE
Proposal time format

### PR DESCRIPTION
**Pull request by dkent600**
* In the proposal closing time countdown, format for 1 day and more: 04d : 12h : 20m, format for less than 1 days: 14h : 45m : 02s
* prevents git from adding CRLF to docker/*.sh files on pull
* adds a little documentation
* fixes a documentation bug

### CODE REVIEW

File: src/components/Shared/Countdown.tsx [hunk 1]
```diff
@@ -113,23 +116,25 @@ class Countdown extends React.Component<IProps, IState> {
...
-            <span>Proposal ends:</span>
+          <span>Proposal ends:</span>
           : " "
         }
-        <span className={css.timeSection}>
-           <strong>{this.addLeadingZeros(countDown.days)}d</strong>
-        </span>
-        &nbsp;:&nbsp;
+        {
+          Number(countDown.days) ? <span><span className={css.timeSection}><strong>{this.addLeadingZeros(countDown.days)}d</strong></span> &nbsp; : &nbsp;</span> : ""
```
Comment (tibetsprague) [inline src/components/Shared/Countdown.tsx]: should already by a Number, why the conversion?

Comment (dkent600) [reply src/components/Shared/Countdown.tsx]: For some reason at the time I was worried it was a string.  I'll remove the 'Number' (after testing, of course).

**Comment by dkent600** [comment]
@jellegerbrandy @tibetsprague Since your approvals, I made some CSS changes to clean up the layout.

File: src/components/Shared/Countdown.tsx (hunk 1 above)
Comment (dkent600) [reply src/components/Shared/Countdown.tsx]: Done

**Comment by dkent600** [comment]
@jellegerbrandy  @tibetsprague  I improved some layout glitches and removed the extraneous `Number` call.  Should be good to go now.

File: src/components/Proposal/ProposalCardContainer.tsx [hunk 2]
```diff
@@ -192,7 +196,7 @@ class ProposalCardContainer extends React.Component<IProps, IState> {
             </div>
           </div>
         </div>
-      </div>
+      </div >
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalCardContainer.tsx]: ? wont this break?

Comment (dkent600) [reply src/components/Proposal/ProposalCardContainer.tsx]: I don't think so, and apparently not.  But I removed the space anyway.

**Pull request merged**